Rubberduck.Resources/Inspections/InspectionInfo.resx

```diff
@@ -364,4 +364,13 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="MissingModuleAnnotationInspection" xml:space="preserve">
     <value>Module attributes are not displayed in the VBE. By adding an annotation, you make these attributes more explicit, and Rubberduck can keep annotations and attributes synchronized.</value>
   </data>
+  <data name="KeywordsUsedAsMemberInspection" xml:space="preserve">
+    <value>A keyword is being used as a member in either an enumeration or an user defined type. That can lead to ambiguous resolution. Condier renaming.</value>
+  </data>
+  <data name="LineContinuationBetweenKeywordsInspection" xml:space="preserve">
+    <value>There are line continuations between keywords. There is no good reason to put it there; consider removing them altogether</value>
+  </data>
+  <data name="NonBreakingSpaceIdentifierInspection" xml:space="preserve">
+    <value>The identiifer contains a non-breaking space which looks very much like just an ordinary space, which obfsucates the code and makes for a confusing experience. Consider using only latin characters for the identifiers.</value>
```

Comment:
The recommendation to use only latin characters is betraying the capabilities of the VBE a bit. I don't think the ducky should explicitly advise against naming variables in the native language of the maintainers, even if that language is Korean or Japanese or any of the other non-latin languages the VBE supports

Reply:
I have to slightly disagree. The problem with non-latin characters is that the code will most likely break when used by someone with different regional windows  settings. The VBE always interprets the code based on the current default code page of windows environment. So, using Japanese characters can break when you send it to someone with an English windows. 

Still, this would be its own inspection. Here, _visible characters_ might be more appropriate.